pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration_test.go

```diff
@@ -115,6 +115,12 @@ func Test_CreateGetRunMigrationsAndRuns(t *testing.T) {
 	delMigResp, err := s.DeleteSession(context.Background(), createResp.UID)
 	require.NoError(t, err)
 	require.NotNil(t, createResp.UID, delMigResp.UID)
+
+	// after deleting the session, the snapshots and resources should not exist anymore.
+	// we check the snapshot for now
+	listRunResp2, err := s.GetMigrationRunList(context.Background(), createResp.UID)
```

Comment:
This unit test also tests the to-be-removed workflow which is a little weird. What do you think about converting this unit test to use the new APIs? The equivalent would be seeding the store with some snapshot results.

Reply:
I think I would prefer to do this in a new PR, when we clean the old func (which I am not 100% sure which ones are) and we can merge this one

Reply:
I'm fine leaving unit test cleanup for later